content/news/2022-03-25_add-storage-dashboard/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Add Storage dashboard from David López
+date: "2022-03-25"
+tease: "New pull request from David López merged: Add Storage dashboard"
+hide_tease: true
+autotoc: false
+tags: ['area/UI-UX', 'kind/feature', 'highlight']
```

Comment:
@dannon do you think we can just stick to the Galaxy tags here?

Reply:
I think using different tags based on the galaxy tags may be better here. I'd remove the `area/` and `kind/` prefixes, as these seem more like technical details, than what users would search for.

Reply:
Have done so, thanks Laila.